content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -174,7 +174,8 @@ different Kubernetes components.
 | `TokenRequestProjection` | `false` | Alpha | 1.11 | 1.11 |
 | `TokenRequestProjection` | `true` | Beta | 1.12 | |
 | `TTLAfterFinished` | `false` | Alpha | 1.12 | |
-| `VolumePVCDataSource` | `false` | Alpha | 1.15 | |
+| `VolumePVCDataSource` | `false` | Alpha | 1.15 | 1.16 |
```

Comment:
Sorry, I think it's inclusive, so should be 1.15

Reply:
Nope, my fault completely, thanks for keeping me honest when I'm rushing.